Modules/_json.c

```diff
@@ -1356,6 +1356,27 @@ _steal_accumulate(_PyUnicodeWriter *writer, PyObject *stolen)
     return rval;
 }
 
+
+PyObject* _create_newline_indent(PyObject* indent, Py_ssize_t indent_level)
+{
+    PyObject* newline_indent = NULL;
+
+    PyObject* _current_indent = PySequence_Repeat(indent, indent_level);
+    if (_current_indent == NULL) {
+        goto end;
```

Comment:
This goto will use `start` before it is defined.

Reply:
Initialise it with `NULL` like `newline_indent`.